Add note on conceallevel option for config (#273)
epwalsh/obsidian.nvim: README.md

If you use `vim-markdown` you'll probably want to disable its frontmatter syntax highlighting (`vim.g.vim_markdown_frontmatter = 1`) which I've found doesn't work very well.

#### Concealing characters

> If you wish to use the formatting concealment features, you will need to have `conceallevel` set to a value that allow it, so either

Reviewer: ```suggestion
If you wish to use the formatting concealment features, you will need to have `conceallevel` set to a value that allows it, for example:
```

> `set conceallevel=1` in viml or `vim.opt.conceallevel = 1` on a lua config.
> 

Reviewer: ```suggestion
`set conceallevel=1` in viml or `vim.opt.conceallevel = 1` in a lua config.
```
